getting-started/getting-help.rst

```diff
@@ -53,7 +53,7 @@ These mailing lists are for questions involving the
 development *of* Python, **not** for development *with* Python.
 
 .. _python-ideas: https://mail.python.org/mailman3/lists/python-ideas.python.org
-.. _python-dev: https://mail.python.org/mailman3/lists/python-dev.python.org/
+.. _python-dev: https://discuss.python.org/c/core-dev/23/
```

Comment:
The text above should also be updated, since it still talks about "mailing lists".

Reply:
Updated check now.